mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -64,6 +65,8 @@ sealed trait CompletionValue:
         printer.tpe(tpe)
       case CompletionValue.Document(_, _, desc) => desc
       case _: CompletionValue.Keyword => ""
+      case fileSysMem @ CompletionValue.FileSystemMember(one, two, three) =>
```

Comment:
I remember the tests were failing until I added this, because the result of the test was the whole `FileSystemMember` with all the fields instead of just the label.

Reply:
```suggestion
      case _: CompletionValue.FileSystemMember => ""
```
maybe then